EssentialFeed/EssentialFeediOS/Feed UI/Controllers/FeedImageCellController.swift

```diff
@@ -11,26 +11,32 @@ public protocol FeedImageCellControllerDelegate {
 }
 
 public final class FeedImageCellController: FeedImageView {
-	private let delegate: FeedImageCellControllerDelegate
+	private let cellControllerDelegate: FeedImageCellControllerDelegate
+	private let selectionHandler: () -> Void
 	private var cell: FeedImageCell?
 	
-	public init(delegate: FeedImageCellControllerDelegate) {
-		self.delegate = delegate
+	public init(cellControllerDelegate: FeedImageCellControllerDelegate, selectionHandler: @escaping () -> Void) {
```

Comment:
The delegate shouldn't have been renamed.

```suggestion
	public init(delegate: FeedImageCellControllerDelegate, selectionHandler: @escaping () -> Void) {
```